src/components/sections/team.jsx

```diff
@@ -1,151 +1,143 @@
+      { name: 'Kimin Lee', picture: KiminPicture, description: '' },
+      { name: 'Sreya Katabathuni', picture: SreyaPicture, description: '' },
+      { name: 'Levi Kaplan', picture: LeviPicture, description: '' }
 
-        ]
-    },
-    {
-        teamName: 'Social/Outreach',
-        teamMembers: [
-            { name: 'Gauri Dandi', picture: GauriPicture, description: 'Team Lead' },
-            { name: 'David Yan', picture: DavidPicture, description: '' },
-            { name: 'Bayden Ibrahim', picture: BaydenPicture, description: '' }
-        ]
-    }
+    ]
+  },
+  {
+    teamName: 'Social/Outreach',
+    teamMembers: [
+      { name: 'Gauri Dandi', picture: GauriPicture, description: 'Team Lead' },
+      { name: 'David Yan', picture: DavidPicture, description: '' },
+      { name: 'Bayden Ibrahim', picture: BaydenPicture, description: '' }
+    ]
+  }
 ];
 
-// Render clickable buttons to render each small team
-const RenderSmallTeamButtons = setSmallTeamClicked => {
-    return smallTeamInfo.map(smallTeam => (
-        <div className="team__small-teams-single-button" onClick={() => setSmallTeamClicked(smallTeam)}>
-            <div style={{ cursor: 'pointer' }}>
-                <SecondaryCta textDescription={smallTeam.teamName} />
-            </div>
+const Team = () => {
+  const [smallTeamClicked, setSmallTeamClicked] = useState(smallTeamInfo[0]);
+  return (
+    <div className='team__container'>
+      <div className='team__content-wrapper'>
+        <div className='team__content'>
+          <a className='theme__header'>OUR TEAM</a>
+          <h2 className='team__space-jam'>
+            Meet this years captains behind HackBeanpot!
+          </h2>
+          <p className='team__description'>
+            The HackBeanpot Core Team has been hard at work bringing HackBeanpot
+            2022 to a galaxy near you! We are a group of 22 driven students from
+            universities in the Boston area who aim to make technical experience
+            and knowledge accessible to anyone who’s interested.
+          </p>
+        </div>
+        {TeamPicture}
+      </div>
+      <div className='team__small-teams'>
+        <div className='team__small-teams-buttons'>
+          <SmallTeamButtons setSmallTeamClicked={setSmallTeamClicked} />
         </div>
-    ));
+        <div className='team__small-team-clicked'>
+          <SmallTeam smallTeam={smallTeamClicked} />
+        </div>
+      </div>
+    </div>
+  );
 };
 
-// Renders current team clicked (if no team has been clicked leadership is shown by default)
-const RenderTeamClicked = smallTeam => {
-    return smallTeam.teamMembers.map(teamMember =>
-        RenderIndividaulTeamMember(
-            teamMember.picture,
-            teamMember.name,
-            teamMember.description
-        )
-    );
+// Render clickable buttons to render each small team
+const SmallTeamButtons = ({ setSmallTeamClicked }) => {
```

Comment:
Is there a difference between passing parameters to functions with {} vs without, 
for example what's the difference between 
const SmallTeamButtons = ({ setSmallTeamClicked }) => { 
and
const RenderSmallTeamButtons = setSmallTeamClicked => {

Reply:
the difference is the first is being destructured, whereas the second is just being named a parameter. here's code that's equivalent to the above, to illustrate the difference:

```javascript
const f1 = (a1) => {
  const setSmallTeamClicked = a1.setSmallTeamClicked;
  // a1 is an object that looks like this: { setSmallTeamClicked: <some value> }
}
const f2 = (setSmallTeamClicked) => {
  // the argument is just named setSmallTeamClicked
}
```